integration_tests/service_a/src/schema.py

```diff
@@ -12,11 +12,17 @@ class FunnyText(ObjectType):
     id = external(Int(required=True))
 
 
+@extend(fields='name')
```

Comment:
Let's also add @extend(fields='id') to another node, to test multiple primary keys logic